test/e2e/components/vl-map-layer.js

```diff
@@ -10,11 +10,11 @@ class VlMapLayer extends VlElement {
   }
 
   async getFeatures() {
-    return this.driver.executeScript(`return arguments[0].layer.getSource().getFeatures();`, this);
+    return this.driver.executeScript(`return arguments[0].features;`, this);
   }
 
   async getFeature(id) {
-    const feature = await this.driver.executeScript(`return arguments[0]._geoJSON.writeFeature(arguments[0].layer.getSource().getFeatures().find((feature) => feature.getId() === ${id}));`, this);
+    const feature = await this.driver.executeScript(`return arguments[0]._geoJSON.writeFeature(arguments[0].features.find((feature) => feature.getId() === ${id}));`, this);
```

Comment:
heeft de vl-map-layer ook niet gewoon ne getFeatureById?

Reply:
Yep, dat is al aangepast in de PR omtrent WFS/WMS/WMTS layers.

Reply:
nee, die heeft dat (nog) niet

Reply:
https://github.com/milieuinfo/webcomponent-vl-ui-map/blob/fix-features-ophalen-van-laag/src/vl-map-layer.js lijn 241

Reply:
beter kijken @wautergu beter kijken

Reply:
in huidige branch heeft die dat dan nog niet...